srcs/app/server/WebServer.cpp

```diff
@@ -26,6 +23,9 @@ void	WebServer::Run() {
 		std::memcpy(&write_set_, &master_set_, sizeof(master_set_));
 		int ready_connections =
 			select(max_sd_ + 1, &read_set_, &write_set_, NULL, NULL);
+		if (ready_connections <= 0) {
+			throw std::runtime_error(std::strerror(errno));
```

Comment:
```
On success, select() and pselect() return the number of file descriptors contained in the three returned descriptor sets (that is, the total  number  of  bits
       that are set in readfds, writefds, exceptfds) which may be zero if the timeout expires before anything interesting happens.  On error, -1 is returned, and er‐
       rno is set to indicate the error; the file descriptor sets are unmodified, and timeout becomes undefined.
```
It appears select may return 0, and still not have failed (errno stay 0, which for strerror(0) would always be"Success" ). I understand when it returns zero we may need to quit the execution anyway, but maybe we could do two different cases?
```
if (ready_connections < 0)
    throw std::runtime_error(std::strerror(errno));
```
and
```
if (ready_connections == 0)
    throw std::runtime_error("Some meaningful error message");
```

Reply:
Well spoted, added.